build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -88,36 +99,37 @@ impl GraalVM {
     pub fn url(&self) -> BoxFuture<'static, Result<Url>> {
         let platform_string = self.platform_string();
         let graal_version = self.graal_version.clone();
+        let graal_version_tag =
+            format!("{}.{}.{}", graal_version.major, graal_version.minor, graal_version.patch);
         let client = self.client.clone();
         async move {
             let repo = CE_BUILDS_REPOSITORY.handle(&client);
-            let release = repo.find_release_by_text(&graal_version.to_string()).await?;
+            let release = repo.find_release_by_text(&graal_version_tag).await?;
             crate::github::find_asset_url_by_text(&release, &platform_string).cloned()
         }
         .boxed()
     }
 
     pub fn platform_string(&self) -> String {
-        let Self { graal_version: _graal_version, java_version, arch, os, client: _client } = &self;
+        let Self { graal_version: _graal_version, arch, os, client: _client } = &self;
         let os_name = match *os {
             OS::Linux => "linux",
             OS::Windows => "windows",
             OS::MacOS => "darwin",
             other_os => unimplemented!("System `{}` is not supported!", other_os),
         };
         let arch_name = match *arch {
-            Arch::X86_64 => "amd64",
+            Arch::X86_64 => "x64",
             // No Graal packages for Apple Silicon.
-            Arch::AArch64 if TARGET_OS == OS::MacOS => "amd64",
+            Arch::AArch64 if TARGET_OS == OS::MacOS => "x64",
             Arch::AArch64 => "aarch64",
             other_arch => unimplemented!("Architecture `{}` is not supported!", other_arch),
         };
-        let java_version = format!("java{}", java_version.0);
-        format!("{PACKAGE_PREFIX}-{java_version}-{os_name}-{arch_name}")
-    }
-
-    pub fn root_directory_name(&self) -> PathBuf {
-        PathBuf::from(format!("{}-{}-{}", PACKAGE_PREFIX, self.java_version, self.graal_version))
+        let java_version = format!(
+            "jdk-{}.{}.{}",
+            _graal_version.major, _graal_version.minor, _graal_version.patch
+        );
```

Comment:
as above

Reply:
Fixed in 57c37ed3f08d7c0b07a14ba9aa6ccaeb8b07c0c5